misc/rhcos-layer/tdx/Containerfile

```diff
@@ -12,7 +12,7 @@ COPY tdx/centos-tdx.repo /etc/yum.repos.d
 
 # Replace libibverbs for qemu-kvm-9.0.0. The RHCOS version is libibverbs-48 
 RUN rpm-ostree override replace https://buildlogs.centos.org/centos/9-stream/BaseOS/x86_64/os/Packages/libibverbs-46.0-1.el9.x86_64.rpm
-RUN rpm-ostree install numactl-libs qemu-kvm-9.0.0
+RUN rpm-ostree install numactl-libs qemu-kvm-tdx
 
 #Install CentOS stream kernel
 RUN rpm-ostree override replace https://buildlogs.centos.org/centos/9-stream/virt/x86_64/tdx-devel/Packages/k/kernel-{,core-,modules-,modules-core-,modules-extra-}5.14.0-480.el9s.x86_64.rpm
```

Comment:
Both the CentOS repo links needs to be changed here as well ?

Reply:
Oh, very nice catch!